src/Http/BaseApplication.php

```diff
@@ -119,22 +123,22 @@ public function getPlugins()
      * @param array $config Configuration options for the plugin
      * @return \Cake\Core\PluginInterface
      */
-    public function makePlugin($name, array $config)
+    protected function makePlugin($name, array $config)
```

Comment:
There is no reason this method needs to be public, and I consider it a mistake that I would like to fix now vs. later. If folks think it is a risky change I'm ok with reverting this.

Reply:
Making is protected sounds good to me.